packages/test/test-gc-sweep-tests/src/dataObjectWithCounter.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DataObject } from "@fluidframework/aqueduct";
+import { assert } from "@fluidframework/common-utils";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { SharedCounter } from "@fluidframework/counter";
+
+/**
+ * DataObjectWithCounter increments a SharedCounter as a way of sending ops.
+ *
+ * The SharedCounter is retrieved via handle
+ */
+const counterKey = "counter";
+export class DataObjectWithCounter extends DataObject {
+    private counter?: SharedCounter;
+    public isRunning: boolean = false;
+    public static get type(): string {
```

Comment:
Can you tell me why is this type needed (I have not looked at how this is currently used in the stress tests)?

Reply:
It's for the registry entry for the factory